### DISCUSSION TITLE
Is it possible to render player as a square/rectangle-shape or etc except a circle?

### DISCUSSION THREAD

**Discussion by minstudio02** [wxyz-abcd/node-haxball#43] [Q&A]
I wonder about that .  thanks a lot for your helpful codes!

**Comment by wxyz-abcd** [comment]
You can write a custom renderer, and render everything however you want. However, basro's renderer is not open-source and I still did not release the default renderer source codes. I'm probably going to do it this week or next week, and then you can change everything as you wish. However, if you want to do it on your own, there is a setRenderer function inside the library and a Renderer structure to follow. (It's exactly like the Plugin interface except the "render" function) Look at rendererTemplate.js in src folder of this repo.

**Comment by wxyz-abcd** [comment]
UPDATE: I have just uploaded the default renderer codes. It's in examples/renderers folder of this repo. You can use our website to make changes and test them while you are playing for fast development.

**Comment by wxyz-abcd** [comment]
UPDATE: I have just implemented square shape player rendering as an option for the default renderer. You can check out its source codes.
